Tribler/Test/Community/Multichain/test_multichain_database.py

```diff
@@ -204,3 +204,27 @@ def test_get_insert_time(self):
         self.assertEquals(time_difference.days, 0)
         self.assertLess(time_difference.seconds, 10,
                         "Difference in stored and retrieved time is too large.")
+
+    def test_database_upgrade(self):
+        self.db.executescript(u"UPDATE option SET value = '1' WHERE key = 'database_version';")
+        self.db.close(commit=True)
+        self.db = None
```

Comment:
I guess these sequences of closing the database can be refactored into a separate method?